kvui.py

```diff
@@ -26,6 +26,9 @@
 if Utils.is_frozen():
     os.environ["KIVY_DATA_DIR"] = Utils.local_path("data")
 
+import platformdirs
+os.environ["KIVY_HOME"] = os.path.join(platformdirs.user_config_dir("Archipelago", False), "kivy")
+
```

Comment:
This fixes the error above. I believe the error happens when %appdata%\Archipelago or ~/.config/Archipelago doesn't exist yet, i.e. kivy does not create all parents

```suggestion
os.makedirs(os.environ["KIVY_HOME"], exist_ok=True)

```